server/infra/database/GrowerAccountRepository.ts

```diff
@@ -341,4 +350,47 @@ export default class GrowerAccountRepository extends BaseRepository<GrowerAccoun
 
     return object.rows;
   }
+
+  async getWalletsCount(filter: GrowerAccountFilter) {
+    const knex = this.session.getDB();
+    const result = await knex
+      .select(
+        knex.raw(`
+        COUNT(DISTINCT(${this.tableName}.id)) AS count
```

Comment:
Don't think there is a need to add distinct, since there are no joins involved

Reply:
Agree, removed